packages/twenty-front/src/modules/billing/components/internal/SubscriptionInfoRowContainer.tsx

```diff
@@ -2,26 +2,29 @@ import { type IconComponent } from 'twenty-ui/display';
 import React from 'react';
 import styled from '@emotion/styled';
 import { useTheme } from '@emotion/react';
+import { t } from '@lingui/core/macro';
 
 type SubscriptionInfoRowContainerProps = {
   Icon: IconComponent;
   label: string;
-  value: React.ReactNode;
+  currentValue: React.ReactNode;
+  nextValue?: React.ReactNode;
 };
 
 const StyledContainer = styled.div`
   align-items: center;
   gap: ${({ theme }) => theme.spacing(1)};
   color: ${({ theme }) => theme.font.color.primary};
-  display: flex;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
 `;
 
 const StyledIconLabelContainer = styled.div`
   align-items: center;
   gap: ${({ theme }) => theme.spacing(1)};
   color: ${({ theme }) => theme.font.color.tertiary};
   display: flex;
-  width: 120px;
+  min-width: 0; /* allow label ellipsis within grid column */
```

Comment:
I'm surprised here, we already have components to handle Ellipsis and we should re-use them

Reply:
TextWithOverflowingToolTip for instance